### DISCUSSION TITLE
Capability to schedule the actions

### DISCUSSION THREAD

**Discussion by viveksinghggits** [kanisterio/kanister#1230] [Ideas]
Per current implementation if we want to run an action from the blueprint we will have to create the actionset resource. But there are use cases where people would want to schedule creation of these actionsets, in order to run the actions periodically.
We can discuss if we really want to add this in kanister or we should just add documentation about how to create respective cronjob resource.
We can also try to add a subcommand in kanctl to generate that cronjob resource.

**Comment by nicolas-g** [comment]
Velero supports the creation of scheduled backups out of the box, having to create a cronjob is not something the normal user will feel comfortable, there should be a least some documentation.

**Reply by viveksinghggits** [reply]
I agree, I will make sure that we talk about this in our next community meeting @nicolas-g.

**Comment by ihcsim** [comment]
Resolved by #1566.

**Comment by nicolas-g** [comment]
In my opinion this should be a feature supported out of the box just like in Velero, people might not know or even want to depend on Argo Workflows
